src/test/java/com/beust/jcommander/DefaultUsageFormatterTest.java

```diff
@@ -11,6 +11,78 @@
 @Test
 public class DefaultUsageFormatterTest {
 
+    private enum TestEnum1 {
+        A, B, C, D
+    }
+
+    private enum TestEnum2 {
+    }
+
+    @Test
+    public void testOutputFormat() {
+        class ArgsTemplate {
+            @Parameter(names = {"--a", "-a"})
+            public int a;
+            @Parameter(names = {"--b", "-b"})
+            public int b = 2;
+            @Parameter(names = {"--c", "-c"}, description = "sets c")
+            public int c;
+            @Parameter(names = {"--d", "-d"}, description = "sets d")
+            public int d = 2;
+            @Parameter(names = {"--e"})
+            public TestEnum1 e;
+            @Parameter(names = {"--f"})
+            public TestEnum1 f = TestEnum1.A;
+            @Parameter(names = {"--g"}, description = "sets g")
+            public TestEnum1 g;
+            @Parameter(names = {"--h"}, description = "sets h")
+            public TestEnum1 h = TestEnum1.A;
+            @Parameter(names = {"-i"})
+            public TestEnum2 i;
+            @Parameter(names = {"-k"}, description = "sets k")
+            public TestEnum2 k;
+        }
+
+        // setup
+        StringBuilder sb = new StringBuilder();
+        JCommander jc = new JCommander(new ArgsTemplate());
```

Comment:
Please use the builder for this, I'd like to completely deprecate the constructor.